tests/test_cli.py

```diff
@@ -0,0 +1,79 @@
+import imp
+import shlex
+import sys
+import threading
+
+import runs
+
+import pytest
+from strip_ansi import strip_ansi
+
+
+def test_cli_version(capfd):
+    """
+    Verify `mqttshark --version` succeeds.
+    """
+    run_mqttshark("--version")
+    out, err = capfd.readouterr()
+    assert out.strip() == "mqttshark version 0.0.1"
+
+
+def test_cli_capture(capfd, mosquitto):
+    """
+    Verify capturing a single MQTT PUBLISH succeeds.
+    """
+    interface_name = "lo"
+    if sys.platform == "darwin":
+        interface_name = "lo0"
+
+    ev = threading.Event()
+    def mqttshark():
+        run_mqttshark(f"-i {interface_name}")
+    t = threading.Thread(target=mqttshark)
+    t.start()
+    ev.wait(0.5)
+
+    command = """mosquitto_pub -t 'testdrive' -m '{"temperature": 42.42}'"""
+    runs.run(command)
+    ev.wait(0.25)
+
+    runs.run("sudo pkill tshark")
+    ev.wait(0.25)
```

Comment:
This is very rough. And it will not scale when adding more test cases covering different scenarios. But it seems to work in 95% of the test runs, modulo some flukes, and I think it will be good enough for a start.

We may keep it in the test suite as a test case solely based on non-Pythonic puppetry, but sure other test cases will approach things more elegantly and efficiently.